<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PREETISH" name="testngprogrammes.DependencyTestNG" tests="5" failures="1" timestamp="20 Feb 2018 10:35:26 GMT" time="0.781" errors="0">
  <testcase name="clicklogin" time="0.000" classname="testngprogrammes.DependencyTestNG"/>
  <testcase name="enterpassword" time="0.001" classname="testngprogrammes.DependencyTestNG"/>
  <testcase name="enteruserid" time="0.778" classname="testngprogrammes.DependencyTestNG">
    <failure type="java.lang.AssertionError" message="expected [gudu1] but found [gudu]">
      <![CDATA[java.lang.AssertionError: expected [gudu1] but found [gudu]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at testngprogrammes.DependencyTestNG.enteruserid(DependencyTestNG.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- enteruserid -->
  <testcase name="browserclose" time="0.001" classname="testngprogrammes.DependencyTestNG"/>
  <testcase name="gmailstart" time="0.001" classname="testngprogrammes.DependencyTestNG"/>
</testsuite> <!-- testngprogrammes.DependencyTestNG -->
